setup.cfg

```diff
@@ -2,4 +2,5 @@
 universal=1
 [flake8]
 max-line-length=120
-ignore: E301, E302, E401, E261, E265, E226, F401, E501
+ignore: F401, W504
+#ignore: E301, E302, E401, E261, E265, E226, F401, E501, W504
```

Comment:
Looks like this stray comment snuck back in.

Reply:
Thanks, fixed it before merging.